packages/mobile/src/fiatExchanges/ProviderOptionsScreen.tsx

```diff
@@ -182,39 +195,72 @@ function ProviderOptionsScreen({ route, navigation }: Props) {
     </View>
   ) : (
     <ScrollView style={styles.container}>
-      <SafeAreaView style={styles.content}>
+      <SafeAreaView>
```

Comment:
I think that we're missing a right margin here:

<img width="368" alt="Screen Shot 2021-06-01 at 17 04 21" src="https://user-images.githubusercontent.com/6062888/120383572-6c085f80-c2fb-11eb-8362-b5c17a42d33a.png">

Also, Ramp and Xanpool are not loading, not sure if that's expected

Reply:
> I think that we're missing a right margin here

Yeah, I'll add it.

> Also, Ramp and Xanpool are not loading, not sure if that's expected

It's expected. Ramp doesn't have a quote endpoint yet and Xanpool will only fetch quotes for users in supported locations (a handful of countries in Asia).